_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -77,7 +78,7 @@ This rule is designed specifically for [2.4.4 Link Purpose (In Context)][sc244],
 
 #### Passed Example 1
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context] and link to the [same resource][].
+These two HTML `a` elements have the same non-empty [accessible name][], empty [accessible description][], the same [context][programmatically determined link context] and resolve to the [same resource][].
```

Comment:
(Minor.)

I suggest that you add a comma i.e. replace this:
the same [context][programmatically determined link context] and resolve to the [same resource][].
with this:
the same [context][programmatically determined link context], and resolve to the [same resource][].

Then the comma usage will match the other passed examples.  And the sentence will be more clear, I think.